packages/cli/src/commonlib/telemetry.ts

```diff
@@ -31,22 +32,40 @@ export class CliTelemetryReporter implements TelemetryReporter {
     if (rootPath) {
       this.rootFolder = rootPath;
       this.reporter.setAppRoot(rootPath);
+
+      // add shared properties
+      const fixedProjectSettings = getFixedCommonProjectSettings(rootPath);
+      this.addSharedProperty(TelemetryProperty.ProjectId, fixedProjectSettings?.projectId ?? "");
+      this.addSharedProperty(
+        TelemetryProperty.IsFromSample,
+        fixedProjectSettings?.isFromSample ?? ""
+      );
+      this.addSharedProperty(
+        TelemetryProperty.ProgrammingLanguage,
+        fixedProjectSettings?.programmingLanguage ?? ""
+      );
+      this.addSharedProperty(TelemetryProperty.HostType, fixedProjectSettings?.hostType ?? "");
```

Comment:
Is user able to change "HostType" after a project is created?

Reply:
Host type will be decide when create a project as spfx or azure. It can not be changed.